yang/frr-filter.yang

```diff
@@ -292,8 +292,6 @@ module frr-filter {
           mandatory true;
 
           case ipv4-prefix {
-            when "../type = 'ipv4'";
```

Comment:
Maybe we should add a comment here?

The conditional is correct, but it causes performance issues right?